transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/internal/ExternalPRequestContext.java

```diff
@@ -0,0 +1,25 @@
+package com.google.android.datatransport.cct.internal;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class ExternalPRequestContext {
```

Comment:
`ExternalPRequestContext` is an odd name, why abbreviate just the P?

Reply:
Discussed offline